src/GregClient/Requests/PackageUpload.cs

```diff
@@ -31,14 +32,9 @@ public override string Path
             get { return "package"; }
         }
 
-        public override Method HttpMethod
+        public override HttpMethod HttpMethod
         {
-            get { return Method.POST; }
-        }
-
-        internal override IList<Parameter> GetParamsToSign(ref RestRequest request)
```

Comment:
@pinzart90 @aparajit-pratap - latest changes - I have removed all these methods relating to actually signing.